cores/esp32/esp32-hal-uart.c

```diff
@@ -30,273 +30,349 @@
 #include "soc/rtc.h"
 #include "esp_intr_alloc.h"
 
-#define UART_REG_BASE(u)    ((u==0)?DR_REG_UART_BASE:(      (u==1)?DR_REG_UART1_BASE:(    (u==2)?DR_REG_UART2_BASE:0)))
-#define UART_RXD_IDX(u)     ((u==0)?U0RXD_IN_IDX:(          (u==1)?U1RXD_IN_IDX:(         (u==2)?U2RXD_IN_IDX:0)))
-#define UART_TXD_IDX(u)     ((u==0)?U0TXD_OUT_IDX:(         (u==1)?U1TXD_OUT_IDX:(        (u==2)?U2TXD_OUT_IDX:0)))
+#define UART_REG_BASE(u)	((u==0)?DR_REG_UART_BASE:(		(u==1)?DR_REG_UART1_BASE:(	  (u==2)?DR_REG_UART2_BASE:0)))
```

Comment:
I don't understand why these are all back, I did not run any formatting tool.

Reply:
it is very much possible that your Editor/IDE does that on it's own